data/user_data_base.ps2

```diff
@@ -199,6 +209,12 @@ function Get-MyName {
     return $nameTag.Value
 }
 
+function Get-Apps {
+    $instanceId = (New-Object System.Net.WebClient).DownloadString("http://169.254.169.254/latest/meta-data/instance-id")
+    $appsTag =  Get-EC2Tag | Where-Object {$_.ResourceId -eq $instanceId -and $_.Key -eq 'Apps'}
```

Comment:
Does this mean we're going to have to have a new apps tag added for all ASGs? if the tag doesn't exist can we default to using hudl-{group}?